--- azimuthalIntegrator.py	2019-03-19 19:15:46.031279325 +0000
+++ azimuthalIntegrator_corrected.py	2019-03-19 19:23:58.904540629 +0000
@@ -175,7 +175,11 @@
     IntegrationMethod(2, "full", "CSR", "cython", old_method="full_csr",
                       class_funct=(splitPixelFullCSR.FullSplitCSR_2d, splitPixelFullCSR.FullSplitCSR_2d.integrate))
 
-from .opencl import ocl
+try:
+    from .opencl import ocl
+except ImportError:
+    ocl = None
+
 if ocl:
     devices_list = []
     devtype_list = []
